doczrc.js

```diff
@@ -13,6 +13,9 @@ export default {
     plugins: [...babelrc.plugins, 'babel-plugin-inline-import']
   }),
   indexHtml: 'public/docz.html',
+  htmlContext: {
+    favicon: 'public/favicon-m.png',
```

Comment:
Just one question, plz: there is any particular reason to name it with `-m` suffix?

What do you think about just `favicon.png`?

```suggestion
    favicon: 'public/favicon.png',
```

Reply:
Fixed ✨